### ISSUE TITLE
Create a section for blocked third-party cookies

### ISSUE THREAD

**Issue by fellyph** [GoogleChromeLabs/ps-analysis-tool#268]
## Feature Request

**Description:**
When I'm using Google Chrome with 3PCD activated, the extension shows a report with the number of blocked cookies and the number of requests blocked.  

**Motivation:**
To give the user clarity of the cookies that are blocked on the cookies' dashboard, for example, in the following image I have one instance without 3PCD enabled and another instance with 3PCD enabled. The number of third-party cookies drops, but I don't have visibility of what has been blocked. 

![Running the report on two chrome instances](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/330792/d26dd86f-adbc-41de-bfcf-101c930f4d77)

On the console, I can get this report:
![Screenshot 2023-11-09 at 20 12 20](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/330792/cb7b4f98-bdd7-4801-a66c-117ee965763b)

It would be nice to have a section in the dashboard with the blocked cookies:
![Screenshot 2023-11-09 at 20 13 07](https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/330792/f3bbf060-7228-400c-924a-c06ee95616c5)

**User Story:**
As a user, I want to see the amount of blocked cookies so that I can have clarity on what was affected by the cookies' deprecation.

**Acceptance Criteria:**
- [ ] ...
- [ ] ...
- [ ] ...

**Additional Information:**
[Include any additional information or context that may be helpful, such as related resources or examples.]

**Comment by amedina** [comment]
This is a good suggestion and it is already in our roadmap. We will be prioritizing it.

**Comment by amedina** [comment]
The "Blocked Cookies" Section in PSAT's cookies landing page will display the following information: 

1. Cookie name
2. Cookie description
3. Blocked reason

An enhancement to the baseline implementation: click on blocked cookie, navigate to frame in PSAT's cookie table, highlight selected cookie.

cc @amovar18 @mayan-000 @ayushnirwal

**Comment by amedina** [comment]
@amovar18 We may want to do this a bit different. Instead of putting all the information above on a table in the Cookie Landing Page, we should follow an approach consistent with the nature of the landing page. 

This means designing a simple UI similar to the Cookie Classification at the top, where we show a counter with the number of blocked cookies, with sub-divisions for the different blocking reasons. And below it, similar to the cookie classification information above, sections with the explanation/description of the blocking rationale for the identified blocked cookies. 

@prasadnevase @mayan-000

**Comment by amovar18** [comment]
@amedina this is how it will look now. Is this what you had imagined.
<img width="1433" alt="Screenshot 2023-12-11 at 2 29 14 PM" src="https://github.com/GoogleChromeLabs/ps-analysis-tool/assets/59614577/98fa68dc-cb35-4f6a-a088-d6ec5acd063a">

**Comment by maitreyie-chavan** [comment]
@amovar18 New section names under which all the information would reside

1. Cookie Classification 
2. Cookie Blockages
3. Frames

**Comment by maitreyie-chavan** [comment]
Addressed and released as a part of [v0.4.0](https://github.com/GoogleChromeLabs/ps-analysis-tool/releases/tag/v0.4.0), closing this issue.
